src/appendices/phenotype.md

```diff
@@ -0,0 +1,261 @@
+| `run`            | CONDITIONAL ; If there are multiple runs within any session | A chronological `run` number is used when a measurement tool or assessment described by a tabular file was repeated within a session. |
+| `acq_time`       | OPTIONAL        | If acquisition time is available, the `acq_time` column CAN be used to record the time of acquisition of each row in the tabular file. |
+
+Furthermore, if you have to add a `session_id` column to the tabular phenotypic data, you then MUST also introduce a session directory to the imaging data, even if only one imaging session has been created. This rule can be considered as "**if anyone uses sessions, everyone uses sessions**." And vice versa, if imaging data has session directories, all imaging data and tabular phenotypic data MUST have sessions.
+
+This produces a file in which same-participant entries can take up as many rows as needed according to the smallest unit of acquisition. The combination of values in the `participant_id`, `session_id`, and `run` (if present) columns MUST be unique for the entire tabular file.
+
+### 4. Sessions file
+
+If there is more than one session for any one participant, then it is REQUIRED to provide a sessions file at the dataset root. The sessions file MUST list all sessions for all subjects across imaging and tabular phenotypic data.
+
+When a sessions file is in use, you MUST NOT provide additional sessions files at the  participant-level which would otherwise use the inheritance principle. If a sessions file is provided, then it MUST begin with a `participant_id` column followed immediately by a `session_id` column. The data dictionary JSON’s `session_id` field MUST include `Levels` with the description of each `session_id`.
+
+### 5. Sessions age
+
+It is RECOMMENDED to use the `age` column to record participant age at every session. This reduces data duplication across tabular data files. The `Units` of `age` do not have to be years so long as the units of the age are written in `sessions.json`. Consider participant privacy or study objectives when selecting the `Units` of `age` or the accuracy of `age` data.
+
+### 6. Sessions `acq_time`
+
+Whenever possible, it is RECOMMENDED to also collect acquisition time for tabular phenotypic data and store the time of acquisition[^2] of each row inside a column named `acq_time` in the sessions file.
+
+When needed to preserve participant privacy, you SHOULD record relative acquisition times with respect to the earliest session. Relative session acquisition times MAY be listed as durations from the earliest session (baseline) in days, months, or years using the `acq_time` column.
+
+### 7. Demographics file
+
+Some studies collect demographics into their own tabular phenotypic data file already. In these cases, it is RECOMMENDED to house this data in the `phenotype/` directory as a TSV called `demographics.tsv` and its corresponding data dictionary JSON called `demographics.json`.
```

Comment:
For furture, maybe we should expand on this to clarify the difference between `participants.tsv` and `demographics.tsv`?